src/libs/auth-utils.ts

```diff
@@ -29,4 +34,64 @@ export const parseBasicAuth = (authorizationHeader?: string): BasicAuth | null =
   }
 
   return null;
+}
+
+/**
+ * Parses bearer auth token from authentication header
+ *
+ * @param authorizationHeader authentication header
+ * @returns bearer auth token
+ */
+export const parseBearerAuth = (authorizationHeader: string | undefined): string | null => {
+  if (!authorizationHeader?.toLocaleLowerCase().startsWith("bearer ")) {
+    return null;
+  }
+
+  const buffer = Buffer.from(authorizationHeader.substring(7), "base64");
```

Comment:
Currently the token is a SHA256 encrypted string, I don't think that's possible to decode it as base64.

Reply:
Changed